exporters/otlp/profiles/build.gradle.kts

```diff
@@ -4,13 +4,35 @@ plugins {
   // id("otel.publish-conventions")
 
   id("otel.animalsniffer-conventions")
+
+  id("com.squareup.wire")
 }
 
 description = "OpenTelemetry - Profiles Exporter"
 otelJava.moduleName.set("io.opentelemetry.exporter.otlp.profiles")
 
+val versions: Map<String, String> by project
 dependencies {
+  protoSource("io.opentelemetry.proto:opentelemetry-proto:${versions["io.opentelemetry.proto"]}")
```

Comment:
You can avoid this by adding `ExportProfilesServiceRequest` to `./exporters/otlp/common/build.gradle.kts`:
```
wire {
  root(
    "opentelemetry.proto.collector.trace.v1.ExportTraceServiceRequest",
    "opentelemetry.proto.collector.metrics.v1.ExportMetricsServiceRequest",
    "opentelemetry.proto.collector.logs.v1.ExportLogsServiceRequest",
    "opentelemetry.proto.collector.logs.v1.ExportLogsServiceRequest",
    "opentelemetry.proto.collector.profiles.v1experimental.ExportProfilesServiceRequest",
  )

  custom {
    schemaHandlerFactoryClass = "io.opentelemetry.gradle.ProtoFieldsWireHandlerFactory"
  }
}
```

Reply:
Hmm. Yes, that's where it belongs in the long run. Putting it there in the short term would pull experimental/alpha bits into the otherwise more stable otlp/common module. I thought adding the separate profiles module was aimed at avoiding that?  Happy to move it if you see some advantage.

Reply:
Moved just the wire auto-gen for now. The marshalers depend on the data interfaces, so common would depend on profiles if we shifted the marshalers but not the data model classes. I'm not wild about having a stable artifact depend on an unstable one, but we can revisit as and when we have a logging exporter or other things using them.